shielder/deploy/deploy.sh

```diff
@@ -64,6 +64,18 @@ prepare_fs() {
   log_progress "✅ Directories are set up"
 }
 
+install_ink_wrapper() {
+  cargo install --git https://github.com/Cardinal-Cryptography/ink-wrapper ink-wrapper
+}
```

Comment:
it is already published so we could install it directly from crates.io (I think); but apart from that, in this deployment script we were used to work with purely dockerized tools, which is way more portable - now the users have to have `cargo` installed (before we were using docker image with `cargo` and `cargo contract`)

it is not a blocker for this PR (it's already big enough), but I would insist on dockerizing ink wrapper - this can be easily done as a workflow in its repository, to publish docker image tagged with crate version; cc @obrok

Reply:
Added an issue for that https://cardinal-cryptography.atlassian.net/browse/A0-2313

Reply:
I don't think it makes sense to publish every small tool like this as a docker image. Instead:

1. We can add `ink-wrapper` to `ink-dev`.
2. You can always make a customized image for your builds that has the set of tooling you need and store the Dockerfile in your repo.

Reply:
no strong opinion on the way to do it, but I'm pretty against using pure `cargo` here

Reply:
If possible, I would go for `1.` as we are already using the `ink-dev` image.